platform-metrics/src/test/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistryTest.java

```diff
@@ -151,9 +152,40 @@ public void testDistributionSummary() {
     assertEquals(1, distribution.count());
     assertEquals(100, distribution.totalAmount());
     assertTrue(
-        Arrays.stream(distribution.takeSnapshot().percentileValues()).map(m -> m.percentile())
-            .collect(
-                Collectors.toList()).containsAll(List.of(0.5, 0.95, 0.99)));
+        Arrays.stream(distribution.takeSnapshot().percentileValues())
+            .map(m -> m.percentile())
+            .collect(Collectors.toList())
+            .containsAll(List.of(0.5, 0.95, 0.99)));
+  }
+
+  @Test
+  public void testCache() throws Exception {
+    initializeCustomRegistry(List.of("testing"));
+
+    Cache<String, Integer> cache = CacheBuilder.newBuilder().maximumSize(10).recordStats().build();
+    cache.put("One", 1);
+    Cache<String, Integer> monitor = PlatformMetricsRegistry.registerCache("my.cache", cache);
+    Callable<Integer> loader =
+        new Callable<Integer>() {
+          @Override
+          public Integer call() throws Exception {
+            return -1;
+          }
+        };
+    // Checking cache values
+    assertEquals(monitor.get("One", loader), 1);
```

Comment:
If i remember correctly, the first argument to assertEquals is the expected value and the second one is the actual value.
So, the arguments should be reversed

Reply:
Ah ! Yes